### ISSUE TITLE
[BUG] ip2int() produces different behavior for dask_cudf

### ISSUE THREAD

**Issue by castellanos-eric-bah** [rapidsai/cudf#5600]
**Describe the bug**
Documentation reports that `ip2int()` converts ip strings to integers.
`ip2int()` produces correct output for `cudf` but not for `dask_cudf`.

**Steps/Code to reproduce bug**
```python
In [1]: import dask_cudf 
 
In [3]: dask_cudf.__version__                                                                                                                                 
Out[3]: '0.14.0a+3912.gcfb1f6b'

In [4]: df = dask_cudf.read_csv("toy.csv")
In [5]: ips = df["ip_src"]   
In [6]: ips.head()  
Out[6]: 
0    127.0.0.1
1      1.1.1.1
2      1.1.1.1
Name: ip_src, dtype: object

In [7]: ips.str.ip2int()
ValueError: Expected iterable of tuples of (name, dtype), got [0, 0]
```

**Expected behavior**
We expected our values for the IP addresses to be converted to integers. Here is the output of the last line of code when using `cudf.read_csv()` instead of `dask_cudf.read_csv()` to read in data.

```python
In [11]: df.ip_src.str.ip2int()                                                                                                                               
Out[11]: [2130706433, 16843009, 16843009]
```

**Environment overview (please complete the following information)**
The above example is more or less copied straight from an ipython session in a fresh 0.14 conda env, created with

`conda env create -c rapidsai-nightly -c nvidia -c conda-forge -c defaults python=3.7 cudatoolkit=10.1`

**Comment by kkraus14** [comment]
@galipremsagar was this already changed in 0.15? Did we expose `ip2int` now that nvstrings is removed?

**Comment by galipremsagar** [comment]
@kkraus14 Yes, we did expose `ip2int` from cudf side, so the above case now works in `0.15`:

```python
>>> import cudf
>>> import dask_cudf 
>>> df = cudf.DataFrame({'a':['127.0.0.1', '1.1.1.1']})
>>> ddf = dask_cudf.from_cudf(df, 2)
>>> ddf
<dask_cudf.DataFrame | 1 tasks | 1 npartitions>
>>> ddf.compute()
           a
0  127.0.0.1
1    1.1.1.1
>>> ddf['a'].head()
0    127.0.0.1
1      1.1.1.1
Name: a, dtype: object
>>> ddf['a'].head().str.ip2int()
0    2130706433
1      16843009
Name: a, dtype: int64
>>> ddf['a'].str.ip2int().compute()
0    2130706433
1      16843009
Name: a, dtype: int64
```

**Comment by kkraus14** [comment]
Closing as fixed in 0.15.
